app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Out data vaccine")
+        if (
+            "wearing_a_mask" not in visitor
+            or visitor["wearing_a_mask"] is False
+        ):
+            raise NotWearingMaskError("All friends must be wear masks")
+        return f"Welcome to {self.name}"

```

Comment:
You are using a dictionary to pass the visitor's data. Although this is not wrong, it would be better to use a Visitor class to encapsulate this data. This would make the code more readable and less prone to errors. For example, you could have a Visitor class with attributes like 'vaccinated', 'vaccine_expiration_date' and 'wearing_mask'. Then, you could pass an instance of this class to the 'visit_cafe' method.

Reply:
it's to much